packages/x-data-grid-pro/src/tests/columns.DataGridPro.test.tsx

```diff
@@ -563,7 +563,7 @@ describe('<DataGridPro /> - Columns', () => {
       render(<Test rows={rows} columns={columns} />);
       await apiRef.current?.autosizeColumns();
       await microtasks();
-      expect(getWidths()).to.deep.equal([155, 177]);
+      expect(getWidths()).to.deep.equal([152, 174]);
```

Comment:
These values need updating due to the letter shift described here https://github.com/mui/mui-x/pull/16962#discussion_r1995346150